### ISSUE TITLE
Frosthaven Support

### ISSUE THREAD

**Issue by fifguy85** [ksokhan/gloom-cards#20]
Just curious if you'd be willing to update the app to support Frosthaven classes as well. I'd be happy to help get the image files (for starting classes and those I've unlocked in our campaign :D).

**Comment by ksokhan** [comment]
Yep. I dont have time to write any code or updates right now unfortunately, but happy to accept a PR!

On a related note: there's another app that has come out thats similar-ish and might be useful: https://www.gloomhavencards.com/
